format.py

```diff
@@ -134,10 +146,10 @@ def format_files(src_files: List[str], extra_files: List[str], nb_jobs: int):
     else:
         run_clang_tidy(src_files)
 
-    print("Adding missing final new lines...")
-    # Adding final new lines is safe to do in parallel and can be applied to all types of files
+    print("Whitespace cleanup...")
```

Comment:
I think it fits with the other messages more if this was verb-y
```suggestion
    print("Cleaning up whitespace...")
```